### PR TITLE
ADEN-3594 Set src and slotName in cust_params

### PR DESCRIPTION
**Pull request by drets**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/video/vastBuilder.js [hunk 1]
```diff
@@ -16,10 +16,13 @@ define('ext.wikia.adEngine.video.vastBuilder', [
 		defaultAdUnit = '/5441/VIDEO_ATG',
 		logGroup = 'ext.wikia.adEngine.video.vastBuilder';
 
-	function getCustomParameters() {
+	function getCustomParameters(slotName, src) {
 		var customParameters = [],
 			pageParams = page.getPageLevelParams();
 
+		pageParams.pos = slotName;
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/video/vastBuilder.js]: s/pageParams/params

**Comment by fraszczakszymon** [comment]
LGTM 👍

**Comment by fraszczakszymon** [comment]
LGTM 👍

**Comment by bulaszenko** [comment]
Checked works good

**Pull request merged**